release_notes/Service_Fabric_ReleaseNotes_91CU4.md

```diff
@@ -0,0 +1,62 @@
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Key Value Store (KVS) | **Brief Description**: The "IntrinsicValueThresholdInBytes" setting controls the size of long values stored in a record. If a value exceeds this size, it is stored as an Long Value ID (LVID) within the record. Presently, the default value is 0 which sets the size to 1024 bytes. However, in the future, the value will be changed to 5120 bytes to enable storage of long values up to approximately 5KB within a record. This change is expected to minimize LVID creation and mitigate the risk of LVID exhaustion. <br>**Solution**: The default value of the IntrinsicValueThresholdInBytes is set to 5120 bytes.
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Application Model | **Brief Description**: Service Fabric has a restriction that allows only one application type with the same name to be provisioned or unprovisioned at any given time. However, if an application type contains illegal path characters in its name or version, it becomes stuck in an unprovisioning loop. This situation not only affects the problematic application type but also causes other application types with the same name to fail during provisioning. <br>**Solution**:  Improvements were made in Service Fabric to address the issue with invalid characters. 
+| **Windows - 9.1.1799.9590<br>Ubuntu 18 - 9.1.1592.1<br>Ubuntu 20 - 9.1.1592.1** | **Feature** | Failover Manager | **Brief Description**:  When MinInstancePercentage is set to 100 and the stateless service has N instances, Service Fabric creates additional instances before taking any instances down for upgrade to ensure a total of N instances remain active throughout the upgrade process. However, during the upgrade, if FailoverManager.RelaxCheckForSafeReplicaCloseCount is set to True, only N-1 instances are guaranteed to be available. On the other hand, if FailoverManager.RelaxCheckForSafeReplicaCloseCount is set to False, the upgrade gets stuck in the preSafetyCheck phase, resulting in timeouts and rollbacks due to the unavailability of N instances <br>**Solution**: Set parameters RelaxCheckForSafeReplicaCloseCount as False and IsStrongMinInstanceCountCheckEnabled as True under FailoverManager section in ClusterManifest to move the upgrade forward.  
```

Comment:
Where are the public docs for this scenario? This seems important to bubble up there and reference it?